webapp/src/components/AssetFilters/AssetFilters.tsx

```diff
@@ -143,34 +144,48 @@ export const AssetFilters = ({
   return (
     <Menu className="filters-sidebar">
       {shouldRenderFilter(AssetFilter.Rarity) ? (
-        <RarityFilter onChange={handleRarityChange} rarities={rarities} />
+        <RarityFilter
+          onChange={handleRarityChange}
+          rarities={rarities}
+          defaultCollapsed={defaultCollapsed?.[AssetFilter.Network] || false}
```

Comment:
i think that if you do this is the same behavior
```suggestion
          defaultCollapsed={!!defaultCollapsed?.[AssetFilter.Network]}
```

Reply:
absolutely! good call, changing it to that one